rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -2,13 +2,13 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman_maintain
-%global confdir foreman_maintain
+%global package_name foreman-maintain
```

Comment:
I don't think the `package_name` actually means what it's used for. Mainly, we wanted to match the executable `foreman-maintain` with the directories in `/etc` and `/var/lib`, but I could probably live with `/etc/foreman_maintain`, unless other ideas how to approach this.